Dockerfiles/agent/Dockerfile

```diff
@@ -68,13 +72,18 @@ RUN if [ -n "$WITH_JMX" ]; then apt-get update \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; fi
 
+# make sure we have recent dependencies
+RUN apt-get update && apt-get upgrade -y --no-install-recommends \
```

Comment:
I'm afraid the `apt-get upgrade -y --no-install-recommends` might make the image size balloon up. Do we have other packages than `util-linux` and `ncurses-bin` to force-upgrade? If so, can we list them in the `apt-get install` step instead?